packages/front-end/components/HomePage/Dashboard.tsx

```diff
@@ -11,23 +11,27 @@ import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
 
 export interface Props {
   experiments: ExperimentInterfaceStringDates[];
+  project: string;
 }
 
-export default function Dashboard({ experiments }: Props) {
+export default function Dashboard({ experiments, project }: Props) {
   const { name } = useUser();
 
   const nameMap = new Map<string, string>();
   experiments.forEach((e) => {
     nameMap.set(e.id, e.name);
   });
+  const northStarExperiments = project
+    ? experiments.filter((e) => e.project === project)
+    : experiments;
 
   return (
     <>
       <div className={"container-fluid dashboard p-3 " + styles.container}>
         <h1>Hello {name}</h1>
         <div className="row">
           <div className="col-md-12">
-            <NorthStar experiments={experiments} />
+            <NorthStar experiments={northStarExperiments} />
```

Comment:
This isn't the experiment list used in the gannt chart.  The `<NorthStarMetricDisplay>` component does it's own API call to get the list of experiments.

Reply:
whoops, fixed (I hope)